client/client_callback.py

```diff
@@ -22,16 +24,31 @@ def __init__(self, job_data=None, cdn_bucket=None, gogs_url=None):
         self.job = TxJob(job_data)
         self.cdn_bucket = cdn_bucket
         self.gogs_url = gogs_url
+        self.tempDir = tempfile.mkdtemp(suffix="", prefix="client_callback_")
```

Comment:
As discussed on Slack, it is helpful for other programmers to at least have all variable names as lowercase with underscore, like the variables above this one. Should be like this in all the code for this PR.